docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -222,4 +223,5 @@ However, on PowerShell you need to use the quotes:
 +
 ----
 PS> $env:HYPERV_VIRTUAL_SWITCH="External (Wireless)"
+PS> setx HYPERV_VIRTUAL_SWITCH %HYPERV_VIRTUAL_SWITCH%
```

Comment:
This is not working in case of `Posh`, It is working in `cmd` .
For some reason PowerShell is not expanding `%HYPERV_VIRTUAL_SWITCH%`
```
C:\Users\CDK\Documents> .\minishift.exe start --iso-url centos
-- Starting profile 'centos'
-- Checking if requested OpenShift version 'v3.7.1' is valid ... OK
-- Checking if requested OpenShift version 'v3.7.1' is supported ... OK
-- Checking if requested hypervisor 'hyperv' is supported on this platform ... OK
-- Checking if Hyper-V driver is installed ... OK
-- Checking if Hyper-V driver is configured to use a Virtual Switch ... OK
-- Checking if user is a member of the Hyper-V Administrators group ... OK
-- Checking the ISO URL ... OK
-- Checking if provided oc flags are supported ... OK
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ..... FAIL E0313 02:53:43.604921     864 start.go:369] Error starting the VM: Error creating the VM. Error with pre-create check: "vswitch \"%HYPERV_VIRTUAL_SWITCH%\" not found". Retrying.
Error starting the VM: Error creating the VM. Error with pre-create check: "vswitch \"%!H(MISSING)YPERV_VIRTUAL_SWITCH%!\(MISSING)" not found"
```

Reply:
That's my mistake -- PowerShell would need to use `$env:HYPERV_VIRTUAL_SWITCH` to expand the environment variable, not `%HYPERV_VIRTUAL_SWITCH%` like in Command Prompt.

This has been rewritten to avoid using shell expansion due to the fact that the environment variable may not be defined when using `setx` for some users.

Reply:
Thanks @robin-owen for the clarification..